src/pages/library/Datasets.js

```diff
@@ -611,10 +611,10 @@ const rareX = () =>
         {
           'aria-label': 'Request access to RareX data',
           tooltip: 'View details on data and request access',
-          href: 'https://rare-x.terra.bio/#workspaces/Rare-x-Terra-Billing/RARE-X',
+          href: 'https://rare-x.terra.bio/#workspaces/rare-x_gg/RAREX_Researcher_Dashboard',
```

Comment:
I tried opening the link and got the `Could not display workspace` message but I assume that's because it's not a public workspace. Unfortunately, unlike what the the tooltip and arial-label say, there's no way to request access to the dataset from that page. It says "please contact the workspace owner" but it doesn't say who that is.